modsecurity.yaml

```diff
@@ -58,6 +58,10 @@ pipeline:
 
   - uses: strip
 
+  - runs: |
+      # We don't need the static libraries
+      find ${{targets.destdir}} -name '*.a' -print | xargs rm
```

Comment:
Could you try using the split/static pipeline to put it into a sub package?

Reply:
Done!